chat.js

```diff
@@ -102,8 +108,11 @@ Chat.multiLinePattern = new PatternTester();
  * Load command files
  *********************************************************/
 
+/** @type {CommandContext} */
```

Comment:
It really isn't though...

Reply:
Oh, those should be `ChatCommands`, right?

Reply:
I think so